bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/calc/SunCalc.java

```diff
@@ -144,6 +144,50 @@ private boolean isSunUpAllDay(Calendar calendar, double latitude, double longitu
         return true;
     }
 
+    /**
+     * Returns the time when the sun is at its minimum elevation
+     */
+    private Calendar getMinSunElevationTime(@Nullable Calendar noon, double latitude, double longitude,
+            @Nullable Double altitude) {
+        Objects.requireNonNull(noon, "A solar noon should always exist");
+
+        Calendar cal = (Calendar) noon.clone();
+        Sun sun = new Sun();
+
+        double minElevation = Double.POSITIVE_INFINITY;
+        Calendar minCal = (Calendar) noon.clone();
+
+        Double previousElevation = null;
+        boolean wasDecreasing = false;
+
+        final int MINUTES_SPAN = 14 * 60; // don't scan 24h, 14 will be enough
+
+        for (int minutes = 0; minutes <= MINUTES_SPAN; minutes += CURVE_TIME_INTERVAL) {
```

Comment:
I think there might be a problem here - I can't see that the loop variable `minutes` is actually used in the loop.

Reply:
Yes. Btw I managed to remove this variable in next PR.